docs/en/deploy/Running-Alluxio-On-Yarn.md

```diff
@@ -6,6 +6,14 @@ group: Deploying Alluxio
 priority: 6
 ---
 
+TODO: Expand the explanation of why we prefer to run alongside YARN instead of on top. Relevant
+mailing list thread: https://groups.google.com/forum/#!topic/alluxio-users/KEvBE9AiJlE
+TODO: Tell users how to configure Alluxio servers running on Yarn
+TODO: Add troubleshooting help
```

Comment:
where would we talk about locality troubleshooting? Is part of this or somewhere else?

Reply:
I think we should handle locality in the compute applications section. The steps for troubleshooting should be the same whether Alluxio is running standalone or on top of YARN/Mesos/etc